src/autopas/CMakeLists.txt

```diff
@@ -7,12 +7,18 @@ ADD_LIBRARY(autopas STATIC
         ${MY_SRC}
         )
 
-find_package(Threads REQUIRED)
-
 target_link_libraries(autopas
         PUBLIC
         rt # required for Time.h
         ${CMAKE_THREAD_LIBS_INIT} # required for Logger and ExceptionHandler
+        # Do not link against openmp when using archer
+        $<$<AND:$<BOOL:${OPENMP}>,$<NOT:$<BOOL:${ARCHER}>>>:OpenMP::OpenMP_CXX>
         )
 
+target_compile_definitions(autopas
```

Comment:
move this to the respective cmake modules

Reply:
only partially possible: e3c1c0f3e68e09a62b69b7b4856fa9010181f9ae